adhocracy-plus/templates/header_lower.html

```diff
@@ -5,36 +5,34 @@
     <div class="col-md-10 offset-md-1">
         <nav class="header-lower__nav">
 
-            {% if ORGANISATION and ORGANISATION.logo %}
-              {% url 'organisation' organisation_slug=ORGANISATION.slug as organisation_url %}
-                <a href="{% url 'organisation' organisation_slug=ORGANISATION.slug %}" class="header-lower__logo-box" rel="home">
-                    <img src="{% thumbnail ORGANISATION.logo '0x160' %}" height="80" alt="{{ ORGANISATION.name }}" class="header-lower__logo-img" />
+            {% if organisation.logo %}
+              {% url 'organisation' organisation_slug=organisation.slug as organisation_url %}
+                <a href="{% url 'organisation' organisation_slug=organisation.slug %}" class="header-lower__logo-box" rel="home">
+                    <img src="{% thumbnail organisation.logo '0x160' %}" height="80" alt="{{ organisation.name }}" class="header-lower__logo-img" />
                 </a>
-            {% elif ORGANISATION %}
+            {% else %}
               <div class="header-lower__brand u-bg-body">
                 <a class="header-lower__brand--link font-weight-bold" href="{% url 'organisation' organisation_slug=ORGANISATION.slug %}">
```

Comment:
so here it should be changed to organisation_slug=organisation.slug, shouldnt it?

Reply:
Oh my, I looked at the wrong line. Yes, exactly, that slipped back in in a rebase! Will fix it! Thank you!